### PR TITLE
feat: add Silex runtime version reporting

### PR DESCRIPTION
**Pull request by tomlongridge**
Uses new hook in Configuration to append the Symfony and Silex application versions to device data to make it reportable.

## Goal

Augments reporting with the version of Symfony and Silex in use when an event occurred in case of issues in a specific version of the framework.

## Design

Added data to device.runtimeVersions.symfony and .silex as per other notifiers.

### Changed

* AbstractServiceProvider.makeClient - appends the Symfony and Silex versions from the $app and HttpKernel classes to the device data through the Bugsnag client

## Tests

* No unit tests present for this level of testing - checked the data was present after running the example silex2 project

## Review

For the submitter, initial self-review:

- [x] Commented on code changes inline explain the reasoning behind the approach
- [x] Reviewed the test cases added for completeness and possible points for discussion
- [x] A changelog entry was added for the goal of this pull request
- [x] Check the scope of the changeset - is everything in the diff required for the pull request?
- This pull request is ready for:
  - [ ] Initial review of the intended approach, not yet feature complete
  - [ ] Structural review of the classes, functions, and properties modified
  - [x] Final review

For the pull request reviewer(s), this changeset has been reviewed for:

- [ ] Consistency across platforms for structures or concepts added or modified
- [ ] Consistency between the changeset and the goal stated above
- [ ] Internal consistency with the rest of the library - is there any overlap between existing interfaces and any which have been added?
- [ ] Usage friction - is the proposed change in usage cumbersome or complicated?
- [ ] Performance and complexity - are there any cases of unexpected O(n^3) when iterating, recursing, flat mapping, etc?
- [ ] Concurrency concerns - if components are accessed asynchronously, what issues will arise
- [ ] Thoroughness of added tests and any missing edge cases
- [ ] Idiomatic use of the language

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
 Changelog
 =========
 
+## TBD
+
+* Add Symfony/Silex version string to report and session payloads (device.runtimeVersions)
```
Comment (mikeewhite) [inline CHANGELOG.md:6]: We don't currently support session tracking in Symfony/Silex

**Review by mikeewhite — APPROVED** [review]
LGTM
